recipes/mercury/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "mercury" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mercury-{{ version }}.tar.gz
+  sha256: cae90d18da56226f1fb28443eba07a6bbf1c04ad4bdd2dc39bc07d9a8d2a2c2b
+
+build:
+  entry_points:
+    - mercury=mercury.mercury:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=50.0.0
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - django ==3.2.5
+    - djangorestframework ==3.12.4
+    - django-filter ==21.1
+    - markdown ==3.3.6
+    - celery ==5.1.2
+    - sqlalchemy ==1.4.27
+    - gevent
+    - nbconvert >=6.5.0
+    - ipython_genutils
+    - pyyaml ==6.0
+    - django-cors-headers ==3.10.1
+    - ipython ==7.30.1
+    - ipykernel ==6.6.0
+    - psutil ==5.8.0
+    - whitenoise ==5.3.0
+    - python-dotenv ==0.19.2
+    - django-drf-filepond ==0.4.1
+    - croniter ==1.3.5
+    - pyppeteer ==1.0.2
+    - channels ==4.0.0
+    - websocket-client ==1.4.2
+    - execnb
+    - ipywidgets ==8.0.3
+
+test:
+  imports:
+    - mercury
+  commands:
+    - pip check
+    - mercury --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mljar/mercury
+  summary: Turn Jupyter Notebook to Web App and share with non-technical users
+  license: Unlicense AND GPL-2.0
```

Looks like this should be:
```suggestion
  license: AGPL-3.0-only OR LicenseRef-Mercury-Commerical
```